domain/transfer/sender_service.go

```diff
@@ -121,20 +107,216 @@ func (s service) CreateNegotiation(ctx context.Context, customerID int, transfer
 			return negotiation, commitErr
 		}
 
-		tokenResponse, err := s.auth.RequestAccessToken(ctx, *customer.Did, organizationDID, ReceiverServiceName, nil)
+		if err = s.sendNotification(ctx, customer, organizationDID); err != nil {
+			// TODO: What to do here? Should we maybe rollback?
+			logrus.Errorf("Unable to notify receiving care organization of updated FHIR task (did=%s): %s", organizationDID, err)
+		}
+	}
+
+	return negotiation, err
+}
+
+func (s service) ConfirmNegotiation(ctx context.Context, customerID int, transferID, negotiationID string) (*domain.TransferNegotiation, error) {
+	// find transfer
+	transfer, err := s.transferRepo.FindByID(ctx, customerID, transferID)
+	if err != nil {
+		return nil, err
+	}
+	if transfer == nil {
+		return nil, fmt.Errorf("transfer with ID: %s, not found", transferID)
+	}
+
+	var (
+		negotiation   *domain.TransferNegotiation
```

Comment:
This value is not set but used on line 149

Reply:
simple fix:

<img width="916" alt="Screenshot 2021-09-08 at 10 33 49" src="https://user-images.githubusercontent.com/703298/132475298-e59e16a1-2bd9-4968-a741-51a626fe4da8.png">